rhv_checks.py

```diff
@@ -338,6 +340,52 @@ def check_vms_distributed_hosts(system, warn=5, crit=10):
         sys.exit(3)
 
 
+def check_hosted_engine_status(system, **kwargs):
+    """ Check the status of all the host's Hosted Engine Status."""
+    okay, critical, unknown, over_all = [], [], [], []
+    command = "hosted-engine --vm-status --json"
+    hosts = system.api.system_service().hosts_service().list()
+
+    for host in hosts:
+        output = ssh_command(host.name, password, command)
+        #Output is in the format of List of JSON removing newline.
+        result = output[0].replace("\\n", "").replace("\n", "")
```

Comment:
Let's check len of `output` before we access index `[0]`

Reply:
`result` we are getting is in a format of `List` with only one element in it. No flaw found during execution.

Reply:
I would still prefer having it checked, in case of empty list we will raise `IndexError` otherwise.

Reply:
okay